app/assets/stylesheets/base.scss

```diff
@@ -7,6 +7,41 @@ body {
   height: 100%;
 }
 
+.navbar-brand-text {
+  padding-left: 5px;
+  margin-top: 7px !important;
+}
+
+.navbar-pf-vertical {
```

Comment:
why not move all of the navigation related css to the top_bar css file? (or maybe we should rename it from top bar to navigation etc).